src/templates/runs/flamegraph.twig

```diff
@@ -72,12 +70,6 @@ d3.json("{{ url('run.flamegraph.data', {id: profile.id|trim }) }}", function(err
         return console.warn(error);
     }
 
-    flameGraph.height(getDepth(data.data) * cellHeight);
-
-    flameGraph.sort(function (a, b) {
```

Comment:
Note to self: This is actually not redundant. Due to the code accessing `data.name` I mistakenly thought this is a name sort, but it's actually looking in XHGui's data object, not the D3 `d.data` object looks similar.

This is from fb8fac599e88d863e33095927a0e6f4d8bb7ff44 and needs to stay.